pkgs/desktops/gnome/extensions/collisions.json

```diff
@@ -40,6 +40,10 @@
       "transparent-window@pbxqdown.github.com",
       "transparentwindows.mdirshad07"
     ],
+    "azan-islamic-prayer-times": [
```

Comment:
You need to handle that collision in `extension-renames.nix`

Reply:
Is there some documentation you can point me to on how to handle this?

Reply:
The basics are documented at the top of the `extension-renames.nix`, let me know if something is still unclear.

Since both extensions are just forks of each other, I suggest using only `azan@faissal.bensefia.id` as it is newer, and setting the other one to `null`.

Reply:
I think I got it right, PTAL.